kodilla-stream/src/main/java/com/kodilla/stream/StreamMain.java

```diff
@@ -1,25 +1,23 @@
 package com.kodilla.stream;
-import com.kodilla.stream.beautifier.PoemBeautifier;
-import com.kodilla.stream.iterate.NumbersGenerator;
+
+import com.kodilla.stream.forumuser.Forum;
+import com.kodilla.stream.forumuser.ForumUser;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class StreamMain {
     public static void main(String[] args) {
 
-        PoemBeautifier poemBeautifier = new PoemBeautifier();
-        poemBeautifier.beautify("Kodilla", (text) -> "|||" + text + "|||");
-        poemBeautifier.beautify("Kodilla", (text) -> "ABC " + text + " CBA");
-        poemBeautifier.beautify("Kodilla", (text) -> "##" + text.toUpperCase() + "##");
-        poemBeautifier.beautify("Kodilla", (text) -> {
-                                                            String result = "";
-                                                            for (int i=0; i<text.length(); i++) {
-                                                                 result = result + text.charAt(i) + "   ";
-                                                            }
-                                                            return "| || |||   " + result + "||| || |";
-                                                            });
-
-        System.out.println("Using Stream to generate even numbers from 1 to 20");
-        NumbersGenerator.generateEven(20);
-    }
+        Forum forum = new Forum();
+        Map<Integer, ForumUser> resultMapOfUsers = forum.getUserList().stream()
+                                                                     .filter(f -> f.getSex()=='M')
+                                                                      .filter(f -> f.getDateOfBirth().getYear()<1997)
+                                                                      .filter(f -> f.getNumberOfPosts()>0)
```

Comment:
"odfiltruje tylko tych użytkowników, którzy mają co najmniej 20 lat"
- twoj program przestanie dzialac prawidlowo w 2018. To jest do poprawy